gpu_support/test_cuda

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Test CUDA
+module use /cvmfs/pilot.eessi-hpc.org/host_injections/nvidia/modules/all/
+module load CUDA
+tmp_dir=$(mktemp -d)
+cp -r $EBROOTCUDA/samples $tmp_dir
+current_dir=$PWD
+cd $tmp_dir/samples/1_Utilities/deviceQuery
+make HOST_COMPILER=$(which g++) -j
+./deviceQuery
+
+if [ $? -eq 0 ] 
+then
+  # Set the color variable
+  green='\033[0;32m'
+  # Clear the color after that
+  clear='\033[0m'
+  echo -e ${green}
+  echo "Congratulations, your GPU is working with EESSI!"
+  echo "  - To build CUDA enabled modules use /cvmfs/pilot.eessi-hpc.org/host_injections/nvidia/ as your EasyBuild prefix"
```

Comment:
If we use a different path, we'll need to tell them to use our EasyBuild hook

Reply:
Actually in general they will need to use the hook as the `gpu` property needs to be added to the module files they build.